CodingStandard/Tests/Classes/ClassCreateInstanceUnitTest.inc

```diff
@@ -3,3 +3,9 @@ $a = new \SomeNS\SomeClass;
 $a = (new \SomeNS\SomeClass)->something();
 $b = new \SomeNS\SomeClass ();
 $b = (new \SomeNS\SomeClass ())->something();
+$b = new self();
```

Comment:
Please rearrange code in a way, that:

* all class instantiations without `()` are assigned to `$a` variable and are grouped together
* all class instantiations where there is a space before `()` are assigned to `$b` variable and are grouped together

This how it's done currently. I bet you haven't noticed that one.